stubs/redis/redis/commands/core.pyi

```diff
@@ -0,0 +1,266 @@
+from collections.abc import Callable, Iterable, Mapping, Sequence
+from datetime import timedelta
+from typing import Any, Generic, TypeVar, Union, overload
+from typing_extensions import Literal
+
+_Value = Union[bytes, float, int, str]
+_Key = Union[str, bytes]
```

Comment:
These probably shouldn't be copy/pasted between `client.pyi` and `commands/core.pyi`. It would be better to have one file import them from the other.